doc-content/enterprise-only/springboot/bus-app-quartz_proc.adoc

```diff
@@ -0,0 +1,103 @@
+org.quartz.threadPool.threadCount = 5
+org.quartz.threadPool.threadPriority = 5
+#============================================================================
+# Configure JobStore
+#============================================================================
+org.quartz.jobStore.misfireThreshold = 60000
+org.quartz.jobStore.class=org.quartz.impl.jdbcjobstore.JobStoreCMT
+org.quartz.jobStore.driverDelegateClass=org.jbpm.process.core.timer.impl.quartz.DeploymentsAwareStdJDBCDelegate
+org.quartz.jobStore.useProperties=false
+org.quartz.jobStore.dataSource=myDS
+org.quartz.jobStore.nonManagedTXDataSource=notManagedDS
+org.quartz.jobStore.tablePrefix=QRTZ_
+org.quartz.jobStore.isClustered=true
+org.quartz.jobStore.clusterCheckinInterval = 5000
+#============================================================================
+# Configure Datasources
+#============================================================================
+org.quartz.dataSource.myDS.connectionProvider.class=org.jbpm.springboot.quartz.SpringConnectionProvider
+org.quartz.dataSource.myDS.dataSourceName=quartzDataSource
+org.quartz.dataSource.notManagedDS.connectionProvider.class=org.jbpm.springboot.quartz.SpringConnectionProvider
+org.quartz.dataSource.notManagedDS.dataSourceName=quartzNotManagedDataSource
+----
+
++
+[NOTE]
+====
+Data source names in the quartz configuration file refer to Spring beans. The connection provider must be set to `org.jbpm.springboot.quartz.SpringConnectionProvider`
+to enable integration with Spring-based data sources.
+====
+
+. Create a managed and an unmanaged data source by adding the following content to the `<business-application>/<business-application>-service/src/main/resources/application.properties` file.
++
+[source, bash]
+----
+# enable to use data base as storage
+jbpm.quartz.db=true
+
+quartz.datasource.name=quartz
+quartz.datasource.username=sa
+quartz.datasource.password=sa
+quartz.datasource.url=jdbc:h2:./target/spring-boot-jbpm;MVCC=true
+quartz.datasource.driver-class-name=org.h2.Driver
+
+# used to configure connection pool
+quartz.datasource.dbcp2.maxTotal=15
+
+# used to initialize quartz schema
+quartz.datasource.initialization=true
+spring.datasource.schema=classpath*:quartz_tables_h2.sql
+spring.datasource.initialization-mode=always
+----
++
+[NOTE]
+====
+The last three lines of the preceding configuration are responsible for automatically initialising the
+database schema. Replace `quartz_tables_h2.sql` with an actual script name.
+====
+
+By default, Quartz requires two data sources:
+
+* Managed data source to participate in the transaction of the {ENGINE}
+* Not managed data source to look up timers to trigger without any transaction handling
+
+Red Hat Business Automation business applications assume that the Quartz database (schema) will be co-located with Process Automation Manager or Decision Manager tables and therefore produce data sources used for transactional operations for Quartz.
```

Comment:
> same data source used for our app is also used for transactional operations for Quartz

that's exactly the thing, if there is only one primary then it will be used for all application data and quartz transactional operations.